Jenkinsfile-publish-image

```diff
@@ -0,0 +1,71 @@
+pipeline {
+    agent {
+        label 'master'
+    }
+    options {
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        DHIS2_VERSION = ''
+        DOCKER_HUB_REPOSITORY = "$DOCKER_HUB_OWNER"
+        DOCKER_IMAGE_NAME = 'core'
+        DOCKER_IMAGE_TAG = ''
+        DOCKER_IMAGE_FULL_NAME = ''
+        COMPOSE_PARAMETER = ''
+    }
+
+    stages {
+        stage('Build image') {
+            steps {
+                script {
+                    pom = readMavenPom file: 'dhis-2/pom.xml'
+
+                    DHIS2_VERSION = pom.version.toLowerCase()
+                    echo "DHIS2 version: ${DHIS2_VERSION}"
+
+                    COMPOSE_PARAMETER = "${env.JOB_NAME}"
+                    echo "Docker compose parameter: ${COMPOSE_PARAMETER}"
+
+                    DOCKER_IMAGE_TAG = "${DHIS2_VERSION}-alpine"
+                    echo "Will tag image as ${DOCKER_IMAGE_TAG}"
+
+                    DOCKER_IMAGE_FULL_NAME = "${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}"
+                    echo "Image full name: ${DOCKER_IMAGE_FULL_NAME}"
+                    
+                    dir("dhis-2/dhis-e2e-test") {
+                        sh "IMAGE_NAME=${DOCKER_IMAGE_FULL_NAME} docker-compose -p ${COMPOSE_PARAMETER} up -d --build"
+                    }   
+                }
+            }
+        }
+
+        stage('Run api tests') {
+            steps {
+                dir("dhis-2/dhis-e2e-test") {
+                    sh "TAG=${DOCKER_IMAGE_TAG} docker-compose -p ${COMPOSE_PARAMETER} -f docker-compose.e2e.yml up --build --exit-code-from e2e-test"
```

Comment:
I'm confused, what is `TAG` here?  It's aliased to `DOCKER_IMAGE_TAG` but used for the tag of a different base image?

Reply:
It's used as a short-hand in the `docker-compose.e2e.yml` file: https://github.com/dhis2/dhis2-core/pull/3309/files#diff-b865f401f8b5994344025e530a5e5b37R6

Reply:
Same as the `IMAGE_NAME` here: https://github.com/dhis2/dhis2-core/pull/3309/files#diff-d56f0e3b867ad4410c433d69c0da0435R13